src/app/modules/angular-slickgrid/filters/autoCompleteFilter.ts

```diff
@@ -310,7 +311,7 @@ export class AutoCompleteFilter implements Filter {
     }
 
     // user might pass his own autocomplete options
-    const autoCompleteOptions = this.columnFilter.filterOptions;
+    const autoCompleteOptions: AutocompleteOption = this.columnFilter.filterOptions;
```

Comment:
this is wrong, it should be `const autoCompleteOptions: AutocompleteOption = this.columnEditor.editorOptions;`